lp-collaborative-workspace/lp-cw-bridge/lp-cw-bridge-transformer/resources/transformation/ado2xwiki.atl

```diff
@@ -216,14 +236,14 @@ rule RootWebHome {
 		className <- 'LPCode.ModelSetClass',
 		number <- 0,
 		wiki <- 'xwiki',
-		space <- thisModule.getModelSetId(),
+		space <- thisModule.modelSet.name,
 		pageName <- 'WebHome',
 		property <- Sequence{msc_p1, msc_p2}
```

Comment:
We can now transform [this](https://github.com/MDEGroup/learnpad/blob/VilniusDecisionV1/lp-collaborative-workspace/lp-cw-bridge/lp-cw-bridge-transformer/resources/model/ado4f16a6bb-9318-4908-84a7-c2d135253dc9.xml#L5676) in the property `documentation` (I've checked, it exists as a property of the object `LPCode.ModelSetClass`).